ni/measurements/metadata/v1/metadata_store_service.proto

```diff
@@ -408,14 +447,15 @@ message RegisterAliasRequest {
   // NOTE: The metadata instance to alias must have already been created
   // in the metadata store in order to register an alias for it.
   oneof alias_target {
```

Comment:
It's a little odd that the `Alias` resource message has a `target_type` enum along with a single value field where as this request message has a `oneof` field instead. Should the `Alias` message use a `oneof` field instead? If not, how would a client make use of the `target_id` field. It seems like they would just have use trial and error to call each GetX RPC until once doesn't return an error.

Reply:
I think the idea is that you would use the `target_type` field to know which `GetX` RPC to call.

However, I would not be opposed to putting the "metadata" messages directly into the Alias message.  I'm pretty sure we thought about doing this originally, but I don't remember details

@hunter-ni ?

Reply:
I'm OK leaving this as is. I just wanted to make sure there was a reason for the difference in the API and that it wasn't just an oversight.